pkg/cmd/flags/logger.go

```diff
@@ -3,42 +3,98 @@ package flags
 import (
 	"fmt"
 	"io"
+	"strings"
+	"time"
+	"unicode"
 
 	"github.com/aquasecurity/tracee/pkg/logger"
 )
 
 func logHelp() string {
-	return `Control logger level priority.
+	return `Control logger options - aggregation and level priority.
 
 Possible options:
-  --log debug               | debug log level
-  --log info                | information log level (default) 
-  --log warn                | warning log level
-  --log error               | error log level
-  --log panic               | panic log level
+  --log aggregate                | turns log aggregation on, delaying output (default is off)
+  --log aggregate:interval       | turns log aggregation on, delaying output to every 'interval' (s, m)
+  --log debug                    | debug log level
+  --log info                     | information log level
+  --log warn                     | warning log level
+  --log error                    | error log level (default)
```

Comment:
Why are we changing the default from info to error? usually info is the default.

Reply:
This has been questioned in the past. @rafaeldtinoco WDYT?

Reply:
I prefer errors only (info is a first level of debugging, that allows end user to check on something without fully debug mode). Lets keep the same for v0.11.1 and discuss for v0.12.0, WDYT ?

Reply:
I think moving the to default ERROR is an exception to most products including ours, where default level is INFO. Rather keep it as is and not move. We don't really use the INFO level much anyway so it's not as if there is some spam issue to resolve by doing this.

Reply:
Well, it all depends on what you consider INFO or ERROR to generalize into "what most apps do" I believe. Either way, someone that has put way more thought into it: https://www.dataset.com/blog/the-10-commandments-of-logging/, and I would say the author is in between ERROR and INFO (with NOTICE). There is something in between that should be the default (considering INFO might scare the user and ERROR might not be enough).

No right and wrong here I believe. @yanivagman FYI on the discussion ^